frontend/src/context/mapStateSlice/index.ts

```diff
@@ -21,6 +22,7 @@ export type MapState = {
   // Note that layerData is mainly for storing vector map data.
   // Tile image loading for raster layer is tracked separately on mapTileLoadingStateSlice
   loadingLayerIds: LayerKey[];
+  relationData: BoundaryRelationsDict;
```

Comment:
let's call this boundaryRelationData maybe? relationData isn't really telling out of context

Reply:
@echaidemenos could you make this change when you have a moment please? Thanks :)